src/lib/command.ts

```diff
@@ -1,9 +1,33 @@
-const UserError = require('./user_error')
+import UserError from './user_error'
+
+type Ctx<P extends boolean> = P extends true ? {
```

Comment:
i tried to not add a lot of generics, but this one here is just super useful for other plugins